.github/workflows/install.sh

```diff
@@ -1,25 +1,17 @@
 set -ex
 
 main() {
-    curl -sSf https://build.travis-ci.org/files/rustup-init.sh | sh -s -- --default-toolchain=nightly -y
-    export PATH=$HOME/.cargo/bin:$PATH
-
-    local target=
-    if [ $TRAVIS_OS_NAME = linux ]; then
-        target=x86_64-unknown-linux-musl
-        sort=sort
-    fi
+    local target=x86_64-unknown-linux-musl
+    local sort=sort
 
     cd $HOME
     git -C binaryen pull || git clone https://github.com/WebAssembly/binaryen binaryen
     cd binaryen
     cmake .
     make wasm-opt
 
-    rustup target install $TARGET
-
-    cargo install -f wasm-gc
-    cargo install -f wasm-bindgen-cli
+    cargo install wasm-gc
+    cargo install wasm-bindgen-cli
 
     # This fetches latest stable release
     local tag=$(git ls-remote --tags --refs --exit-code https://github.com/japaric/cross \
```

Comment:
Why are we even installing cross? That's from way back when we used emscripten. I believe we can throw this out entirely too.